### PR TITLE
feat: :art: add google button

### PR DESCRIPTION
**Pull request by AdrianValdes**
Hi man! 

I added the design for the login google button. I created a `svgs` folder inside the components folder. I got the idea for a dude in github, I think his user is @paschalidi :D. In any case, let my know if that works for our case.

### CODE REVIEW

File: src/webapp/src/components/Navbar/navbar.style.ts [hunk 1]
```diff
@@ -8,7 +9,7 @@ export const HeaderStyles = styled.div`
   width: 100%;
   max-width: 100%;
   box-shadow: inset 0 -1px 0 0 rgba(0, 0, 0, 0.1);
-  min-height: 64px;
+  min-height: ${navbarConstants.height};
```
Comment (paschalidi) [inline src/webapp/src/components/Navbar/navbar.style.ts:12]: 👍

File: src/webapp/src/auth/AuthenticationOptions.tsx [hunk 2]
```diff
@@ -54,6 +55,23 @@ const FacebookButton = styled(Button)`
...
+    width: 320px;
+    color: #4285f4;
+    span {
+      position: absolute;
+      inset: 0px auto 0px 22px;
+      width: 18px;
+    }
+    &:hover {
+      box-shadow: 0 0 6px #4285f4;
+    }
```
Comment (paschalidi) [inline src/webapp/src/auth/AuthenticationOptions.tsx:71]: I see that the other button isnt having any shadow, perhaps for consistency we remove this shadow as well?

**Review by paschalidi — COMMENTED** [review]
Looks good one minor comment!

**Comment by AdrianValdes** [comment]
Look what I found watching a workshop for the second time to review some ideas: 

image

Good that I just needed you repeating it 4 times more :D

**Review by paschalidi — APPROVED** [review]

**Pull request merged**